### PR TITLE
port missing interfaces from ROS1 common_msgs

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Connects to https://github.com/ros2/common_interfaces/issues/27
Closes https://github.com/ros2/common_interfaces/issues/27

This PR doesn't include `SetBool.srv` from `std_srvs` to follow the "services names should carry semantic meaning" guideline

### CODE REVIEW

**Comment by dirk-thomas** [comment]
I would include `std_srvs/SetBool.srv`. While we don't recommend users to rely on it in their API (the same for a lot of other messages in `std_msgs`) this interface is convenient for prototyping and testing.

**Comment by mikaelarguedas** [comment]
Fair enough https://github.com/ros2/common_interfaces/pull/28/commits/fbad6e210736c220fd7d6650eae0472a50e0176e

I'm keeping this PR in backlog because I'm not sure we want to increase build time for messages we dont use yet

File: diagnostic_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@ set(msg_files
 )
 set(srv_files
   "srv/SelfTest.srv"
+  "srv/AddDiagnostics.srv"
```
Comment (dirk-thomas) [inline diagnostic_msgs/CMakeLists.txt]: Order.

**Comment by mikaelarguedas** [comment]
Good catch, thanks

**Comment by mikaelarguedas** [comment]
@ros2/dev planning on merging this given that I expect `nav_msgs` to be reenabled soon and that the additional build time will not be  significant

**Comment by Karsten1987** [comment]
+1 for enabling nav_msgs

**Pull request merged**